### PR TITLE
[CE Sprint #16] [CLOUD-2398] Include the openssl rpm (and binary) in the RH-SSO for OpenShift image

### PR DESCRIPTION
**Pull request by iankko**
<br/>
So we can utilize the OpenShift's serving x509 certificate secrets service to autogenerate keystores and truststore for the RH-SSO server<br>
<br/>
Signed-off-by: Jan Lieskovsky <jlieskov@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[CLOUD-XYA] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Comment by iankko** [comment]
@rcernich @goldmann @ASzc PTAL

File: image.yaml [hunk 1]
```diff
@@ -148,7 +148,7 @@ packages:
           - python-requests
           - python-enum34
           - PyYAML
-
+          - openssl
```
Comment (rcernich) [inline image.yaml:151]: Please remove this and add it to the module that requires it.  Modules should be self contained and not require additional configuration on the part of consumers.

**Review by rcernich — CHANGES_REQUESTED** [review]

File: image.yaml (hunk 1 above)
Comment (iankko) [reply image.yaml:151]: Thanks, Rob. Looking.

**Comment by iankko** [comment]
Closing this one. The change originally covered within this PR is now part of https://github.com/jboss-openshift/cct_module/pull/221.

**Pull request closed without merging**